articles/quickstart/webapp/rails/files/concern.md

```diff
@@ -0,0 +1,18 @@
+---
+name: app/controllers/concerns/secured.rb
```

Comment:
I dont know much about Ruby, but would it be a problem if the name would be just `secured.rb` ? Would that confused readers?

Asking because it would keep things a bit tighter in the UI in terms of filenames.

Reply:
Personally, I believe specifying the path falls close to the tutorial side of things. When sharing the snippets to integrate Auth0, we shouldn't really care much about where the user puts the code.

I do understand it's a small line and I can see how it can still be helpful to mention the full path somewhere. On top of that, as I don't know Ruby so I don't know how much `app/controllers/concerns` is a required path.

Reply:
I'll remove it 👍🏻